<launch>
	<node pkg = "rosaria" type = "RosAria" name = "RosAria">
		<param name="port" value="/dev/ttyS0" type="string"/>
	</node>

	<node pkg = "rosaria_client" type = "teleop" name = "RosAria_teleop">
		<remap from="RosAria/cmd_vel" to="RosAria/cmd_vel_hp"/>
	</node>
	
	<node pkg = "rosaria_client" type = "handpoint" name = "RosAria_handpoint" launch-prefix = "xterm -e"/>

</launch>
